js/pages/concept-sets/concept-manager.js

```diff
@@ -28,12 +28,13 @@ define([
 			this.commonUtils = commonUtils;
 			this.sourceCounts = ko.observableArray();
 			this.loadingSourceCounts = ko.observable(false);
-			this.loadingRelated = ko.observable(false);
+			this.loadingRelated = ko.observable(true);
 
 			this.currentConceptId = params.model.currentConceptId;
 
 			this.subscriptions.push(
 				this.currentConceptId.subscribe((value) => {
+					this.loadingRelated(true);
```

Comment:
Why have you moved the loading flag out of the appropriate function? Why haven't you put it as a first line in the `loadRecordCounts`?